gslib/tests/testcase/base.py

```diff
@@ -201,6 +201,7 @@ def CreateTempFile(self, tmpdir=None, contents=None, file_name=None,
     if isinstance(file_name, (six.text_type, six.binary_type)):
       fpath = os.path.join(tmpdir, file_name)
     else:
+      file_name = map(six.ensure_str, file_name)
```

Comment:
Per the style guide, try to use list/dict/tuple comprehensions instead of `map()`, `filter()`, etc.

http://google.github.io/styleguide/pyguide.html#215-deprecated-language-features

Reply:
It looks like the decision favors using `map()`: http://google.github.io/styleguide/pyguide.html#2152-decision

Joe and I were actually talking about this earlier. :slightly_smiling_face:

Reply:
Ya know, I instinctively looked for that section in the style guide, read the first paragraph, and then linked it. Didn't bother to read the code snippets 🙃 They used to discourage these completely in favor of list comprehensions, but I guess it changed at some point. If the style guide doesn't complain, I won't either :)